builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -278,6 +278,15 @@ def loadServiceConfigurationData() {
 			sh "sed -i -- 's/{conf-region}/${region}/g' ./config/dev-config.json"
 			sh "sed -i -- 's/{conf-region}/${region}/g' ./config/stg-config.json"
 			sh "sed -i -- 's/{conf-region}/${region}/g' ./config/prod-config.json"
+
```

Comment:
why is this change as part of this PR?

Reply:
I saw the jazz admin name of the jazz_services is written from the installer. As we are maintaining the service configuration during the deployment of the jazz core services, I think we dont want that part in installer. Thats why I had removed that from installer. So that we can reduce the jazz core dependency.